Formula/dtrx.rb

```diff
@@ -1,27 +0,0 @@
-class Dtrx < Formula
```

Comment:
Why was this removed? As far as I can tell `dtrx` doesn't have a hard dependency on `unrar`, but can use it when it's in `PATH`.

Reply:
- It is not actively maintained
- Its homepage is gone
- It does not build on Big Sur
- It had an non optional dependency on unrar

It could be accepted back if it fit our criteria: https://docs.brew.sh/Acceptable-Formulae

Reply:
The documentation didn't specify that, but you can re-add the formula if that's the case

Reply:
>As far as I can tell dtrx doesn't have a hard dependency on `unrar`

This is rather difficult to ascertain when it doesn't even have a working homepage